cgltf.h

```diff
@@ -6267,6 +6488,29 @@ static int cgltf_parse_json_root(cgltf_options* options, jsmntok_t const* tokens
 						}
 					}
 				}
+				else if (cgltf_json_strcmp(tokens+i, json_chunk, "KHR_xmp_json_ld") == 0)
+				{
+					++i;
+
+					CGLTF_CHECK_TOKTYPE(tokens[i], JSMN_OBJECT);
+
+					int data_size = tokens[i].size;
+					++i;
+
+					for (int m = 0; m < data_size; ++m) {
+						CGLTF_CHECK_KEY(tokens[i]);
+
+						if (cgltf_json_strcmp(tokens + i, json_chunk, "packets") == 0) {
+							i = cgltf_parse_xmp_json_ld_packets(options, tokens, i + 1, json_chunk, out_data);
+						} else {
```

Comment:
Please fix bracing style here and everywhere else in the patch to conform to the style - braces should be on their own line.

Reply:
sure. will do.